
R version 4.4.2 (2024-10-31) -- "Pile of Leaves"
Copyright (C) 2024 The R Foundation for Statistical Computing
Platform: x86_64-pc-linux-gnu

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type 'license()' or 'licence()' for distribution details.

R is a collaborative project with many contributors.
Type 'contributors()' for more information and
'citation()' on how to cite R or R packages in publications.

Type 'demo()' for some demos, 'help()' for on-line help, or
'help.start()' for an HTML browser interface to help.
Type 'q()' to quit R.

> library(testthat)
> library(packrat)
> 
> test_check("packrat")
Initializing packrat project in directory:
- "/tmp/RtmpWrL4m2/sated"
[ FAIL 17 | WARN 10 | SKIP 50 | PASS 53 ]

══ Skipped tests (50) ══════════════════════════════════════════════════════════
• On CRAN (50): 'test-aaa.R:2:3', 'test-bundle.R:3:3', 'test-bundle.R:25:3',
  'test-cache.R:3:3', 'test-cache.R:75:3', 'test-dependencies.R:4:3',
  'test-dependencies.R:10:3', 'test-dependencies.R:35:3',
  'test-dependencies.R:41:3', 'test-dependencies.R:47:3',
  'test-dependencies.R:53:3', 'test-dependencies.R:59:3',
  'test-dependencies.R:65:3', 'test-dependencies.R:80:3',
  'test-dependencies.R:95:3', 'test-dependencies.R:103:3',
  'test-dependencies.R:109:3', 'test-dependencies.R:126:5',
  'test-dependencies.R:137:5', 'test-dependencies.R:148:5',
  'test-downloader.R:27:3', 'test-downloader.R:45:3', 'test-git.R:2:5',
  'test-hash.R:2:3', 'test-hash.R:11:3', 'test-packrat.R:13:5',
  'test-packrat.R:28:5', 'test-packrat.R:44:5', 'test-packrat.R:57:5',
  'test-packrat.R:71:5', 'test-packrat.R:99:5', 'test-packrat.R:127:5',
  'test-packrat.R:141:5', 'test-packrat.R:153:5', 'test-packrat.R:176:5',
  'test-packrat.R:208:5', 'test-packrat.R:249:5', 'test-packrat.R:271:5',
  'test-packrat.R:276:5', 'test-packrat.R:283:5', 'test-packrat.R:302:5',
  'test-pkg.R:4:3', 'test-pkg.R:10:3', 'test-pkg.R:16:3', 'test-pkg.R:22:3',
  'test-rmarkdown.R:2:3', 'test-rmarkdown.R:16:3', 'test-shiny.R:2:3',
  'test-shiny.R:14:3', 'test-with_extlib.R:2:3'

══ Failed tests ════════════════════════════════════════════════════════════════
── Error ('test-bitbucket.R:15:3'): bitbucketArchiveUrl returns the correct URL ──
<packageNotFoundError/error/condition>
Error in `loadNamespace(x)`: there is no package called 'mockery'
Backtrace:
    ▆
 1. └─base::loadNamespace(x) at test-bitbucket.R:15:3
 2.   └─base::withRestarts(stop(cond), retry_loadNamespace = function() NULL)
 3.     └─base (local) withOneRestart(expr, restarts[[1L]])
 4.       └─base (local) doWithOneRestart(return(expr), restart)
── Error ('test-bitbucket.R:27:3'): bitbucketDownload calls renvDownload in the expected context ──
<packageNotFoundError/error/condition>
Error in `loadNamespace(x)`: there is no package called 'mockery'
Backtrace:
    ▆
 1. └─base::loadNamespace(x) at test-bitbucket.R:27:3
 2.   └─base::withRestarts(stop(cond), retry_loadNamespace = function() NULL)
 3.     └─base (local) withOneRestart(expr, restarts[[1L]])
 4.       └─base (local) doWithOneRestart(return(expr), restart)
── Error ('test-bitbucket.R:42:3'): bitbucketDownload calls bitbucketDownloadHttr in the expected context ──
<packageNotFoundError/error/condition>
Error in `loadNamespace(x)`: there is no package called 'mockery'
Backtrace:
    ▆
 1. └─base::loadNamespace(x) at test-bitbucket.R:42:3
 2.   └─base::withRestarts(stop(cond), retry_loadNamespace = function() NULL)
 3.     └─base (local) withOneRestart(expr, restarts[[1L]])
 4.       └─base (local) doWithOneRestart(return(expr), restart)
── Error ('test-bitbucket.R:60:3'): bitbucketDownload calls downloadWithRetries in the expected contexts ──
<packageNotFoundError/error/condition>
Error in `loadNamespace(x)`: there is no package called 'mockery'
Backtrace:
    ▆
 1. └─base::loadNamespace(x) at test-bitbucket.R:60:3
 2.   └─base::withRestarts(stop(cond), retry_loadNamespace = function() NULL)
 3.     └─base (local) withOneRestart(expr, restarts[[1L]])
 4.       └─base (local) doWithOneRestart(return(expr), restart)
── Error ('test-downloader.R:73:3'): renvDownload calls renv$download, passing in the values it received ──
<packageNotFoundError/error/condition>
Error in `loadNamespace(x)`: there is no package called 'mockery'
Backtrace:
    ▆
 1. └─base::loadNamespace(x) at test-downloader.R:73:3
 2.   └─base::withRestarts(stop(cond), retry_loadNamespace = function() NULL)
 3.     └─base (local) withOneRestart(expr, restarts[[1L]])
 4.       └─base (local) doWithOneRestart(return(expr), restart)
── Error ('test-env.R:24:3'): On Unix, use tar on the path if it exists ────────
<packageNotFoundError/error/condition>
Error in `loadNamespace(x)`: there is no package called 'mockery'
Backtrace:
    ▆
 1. └─base::loadNamespace(x) at test-env.R:24:3
 2.   └─base::withRestarts(stop(cond), retry_loadNamespace = function() NULL)
 3.     └─base (local) withOneRestart(expr, restarts[[1L]])
 4.       └─base (local) doWithOneRestart(return(expr), restart)
── Error ('test-env.R:42:3'): On Unix, use 'internal' as a fallback if no tar is found on the PATH ──
<packageNotFoundError/error/condition>
Error in `loadNamespace(x)`: there is no package called 'mockery'
Backtrace:
    ▆
 1. └─base::loadNamespace(x) at test-env.R:42:3
 2.   └─base::withRestarts(stop(cond), retry_loadNamespace = function() NULL)
 3.     └─base (local) withOneRestart(expr, restarts[[1L]])
 4.       └─base (local) doWithOneRestart(return(expr), restart)
── Error ('test-env.R:67:3'): On Windows, use the system tar if it exists ──────
<packageNotFoundError/error/condition>
Error in `loadNamespace(x)`: there is no package called 'mockery'
Backtrace:
    ▆
 1. └─base::loadNamespace(x) at test-env.R:67:3
 2.   └─base::withRestarts(stop(cond), retry_loadNamespace = function() NULL)
 3.     └─base (local) withOneRestart(expr, restarts[[1L]])
 4.       └─base (local) doWithOneRestart(return(expr), restart)
── Error ('test-env.R:94:3'): On Windows, use 'internal' as a fallback if system tar doesn't exist ──
<packageNotFoundError/error/condition>
Error in `loadNamespace(x)`: there is no package called 'mockery'
Backtrace:
    ▆
 1. └─base::loadNamespace(x) at test-env.R:94:3
 2.   └─base::withRestarts(stop(cond), retry_loadNamespace = function() NULL)
 3.     └─base (local) withOneRestart(expr, restarts[[1L]])
 4.       └─base (local) doWithOneRestart(return(expr), restart)
── Error ('test-github.R:32:3'): githubArchiveUrl returns the correct URL ──────
<packageNotFoundError/error/condition>
Error in `loadNamespace(x)`: there is no package called 'mockery'
Backtrace:
    ▆
 1. └─base::loadNamespace(x) at test-github.R:32:3
 2.   └─base::withRestarts(stop(cond), retry_loadNamespace = function() NULL)
 3.     └─base (local) withOneRestart(expr, restarts[[1L]])
 4.       └─base (local) doWithOneRestart(return(expr), restart)
── Error ('test-github.R:49:3'): githubDownload calls renvDownload in the expected context ──
<packageNotFoundError/error/condition>
Error in `loadNamespace(x)`: there is no package called 'mockery'
Backtrace:
    ▆
 1. └─base::loadNamespace(x) at test-github.R:49:3
 2.   └─base::withRestarts(stop(cond), retry_loadNamespace = function() NULL)
 3.     └─base (local) withOneRestart(expr, restarts[[1L]])
 4.       └─base (local) doWithOneRestart(return(expr), restart)
── Error ('test-github.R:64:3'): githubDownload calls githubDownloadHttr in the expected context ──
<packageNotFoundError/error/condition>
Error in `loadNamespace(x)`: there is no package called 'mockery'
Backtrace:
    ▆
 1. └─base::loadNamespace(x) at test-github.R:64:3
 2.   └─base::withRestarts(stop(cond), retry_loadNamespace = function() NULL)
 3.     └─base (local) withOneRestart(expr, restarts[[1L]])
 4.       └─base (local) doWithOneRestart(return(expr), restart)
── Error ('test-github.R:82:3'): githubDownload calls downloadWithRetries in the expected contexts ──
<packageNotFoundError/error/condition>
Error in `loadNamespace(x)`: there is no package called 'mockery'
Backtrace:
    ▆
 1. └─base::loadNamespace(x) at test-github.R:82:3
 2.   └─base::withRestarts(stop(cond), retry_loadNamespace = function() NULL)
 3.     └─base (local) withOneRestart(expr, restarts[[1L]])
 4.       └─base (local) doWithOneRestart(return(expr), restart)
── Error ('test-gitlab.R:28:3'): gitlabArchiveUrl returns the correct URL ──────
<packageNotFoundError/error/condition>
Error in `loadNamespace(x)`: there is no package called 'mockery'
Backtrace:
    ▆
 1. └─base::loadNamespace(x) at test-gitlab.R:28:3
 2.   └─base::withRestarts(stop(cond), retry_loadNamespace = function() NULL)
 3.     └─base (local) withOneRestart(expr, restarts[[1L]])
 4.       └─base (local) doWithOneRestart(return(expr), restart)
── Error ('test-gitlab.R:44:3'): gitlabDownload calls renvDownload in the expected context ──
<packageNotFoundError/error/condition>
Error in `loadNamespace(x)`: there is no package called 'mockery'
Backtrace:
    ▆
 1. └─base::loadNamespace(x) at test-gitlab.R:44:3
 2.   └─base::withRestarts(stop(cond), retry_loadNamespace = function() NULL)
 3.     └─base (local) withOneRestart(expr, restarts[[1L]])
 4.       └─base (local) doWithOneRestart(return(expr), restart)
── Error ('test-gitlab.R:59:3'): gitlabDownload calls gitlabDownloadHttr in the expected context ──
<packageNotFoundError/error/condition>
Error in `loadNamespace(x)`: there is no package called 'mockery'
Backtrace:
    ▆
 1. └─base::loadNamespace(x) at test-gitlab.R:59:3
 2.   └─base::withRestarts(stop(cond), retry_loadNamespace = function() NULL)
 3.     └─base (local) withOneRestart(expr, restarts[[1L]])
 4.       └─base (local) doWithOneRestart(return(expr), restart)
── Error ('test-gitlab.R:77:3'): gitlabDownload calls downloadWithRetries in the expected contexts ──
<packageNotFoundError/error/condition>
Error in `loadNamespace(x)`: there is no package called 'mockery'
Backtrace:
    ▆
 1. └─base::loadNamespace(x) at test-gitlab.R:77:3
 2.   └─base::withRestarts(stop(cond), retry_loadNamespace = function() NULL)
 3.     └─base (local) withOneRestart(expr, restarts[[1L]])
 4.       └─base (local) doWithOneRestart(return(expr), restart)

[ FAIL 17 | WARN 10 | SKIP 50 | PASS 53 ]
Error: Test failures
Execution halted
